contrib/cirrus/logcollector.sh

```diff
@@ -32,6 +32,14 @@ case $1 in
     df) showrun df -lhTx tmpfs ;;
     ginkgo) showrun cat $CIRRUS_WORKING_DIR/test/e2e/ginkgo-node-*.log ;;
     journal) showrun journalctl -b ;;
+    varlink)
+       if [[ "$TEST_REMOTE_CLIENT" != "true" ]]
```

Comment:
I find positive tests much more readable and friendly. Would you consider changing this one to `==` form?